test/snippet/std/view/subrange.cpp

```diff
@@ -0,0 +1,16 @@
+#include <seqan3/std/view/subrange.hpp>
+#include <seqan3/alphabet/nucleotide/dna4.hpp>
+#include <seqan3/range/view/to_char.hpp>
+#include <seqan3/io/stream/debug_stream.hpp>
+
+using namespace seqan3;
+
+int main()
+{
+//! [example]
+dna4_vector s{"ACTTTGATAN"_dna4};
```

Comment:
There is a `N` in a dna4 vector?

Reply:
It automatically is converted to an A by default. I think this was intentional, since the comment also shows that it gets converted to an A? Or should I just get rid of the N.

Reply:
Since the `N` to `A` conversion of dna4 has nothing to do with the subrange view I would get rid of the N because it might be confusing if you aren't accustomed to our alphabets.